samples/dnn/js_face_recognition.html

```diff
@@ -12,27 +12,40 @@
 
 //! [Run face detection model]
 function detectFaces(img) {
-  var blob = cv.blobFromImage(img, 1, {width: 192, height: 144}, [104, 117, 123, 0], false, false);
```

Comment:
I propose to backport JS part to 4.x.

Reply:
This change in JS sample is moved to this PR: https://github.com/opencv/opencv/pull/25468